rules/mech-building/planning.md

```diff
@@ -5,48 +5,98 @@ permalink: /rules/build-planning/
+| Ethanol Boiler        | Steam      | Ethanol     |
+| Ethanol Engine        | Mechanical | Ethanol     |
+| Ethanol Fuel Cell     | Electrical | Ethanol     |
+| Hydrogen Boiler       | Steam      | Hydrogen    |
+| Hydrogen Engine       | Mechanical | Hydrogen    |
+| Hydrogen Fuel Cell    | Electrical | Hydrogen    |
+| Lithium Ion Regulator | Electrical | Lithium Ion |
+| Methane Boiler        | Steam      | Methane     |
+| Methane Engine        | Mechanical | Methane     |
+| Nuclear Boiler        | Steam      | Nuclear     |
+| Nuclear Generator     | Mechanical | Nuclear     |
+| Nuclear Turbine       | Electrical | Nuclear     |
+
+{.table}
+
+#### Fuel Tank Capacity and Depletion
+Each day, your mech will use up an amount of fuel equal to its Used Power. If you run out of fuel, your mech will be unable to move or use attachments until refueled. 
+
+Some fuel types are much more energy-dense than others, which means that mechs using those types of fuel will typically need to refuel less often. The table below shows how much Power can be stored in small and large Fuel Tanks of each type.
+
+| Fuel Tank          | Fuel        | Capacity (Small) | Capacity (Large) |
+| ------------------ | ----------- | ---------------- | ---------------- |
+| Ethanol Tank       | Ethanol     | 10               | 20               |
+| Hydrogen Tank      | Hydrogen    | 20               | 40               |
+| Lithim Ion Battery | Lithium Ion | 30               | 60               |
+| Methane Tank       | Methane     | 40               | 80               |
+| Reactor Vessel     | Nuclear     | 60               | 120              |
+
+{.table}
+
+Depending on how much power your mech uses, you may need to consider a larger fuel tank or even a different fuel type. 
 
 ### Step by step Mech-Building Outline
 
 1. Develop a vision for your mechaneer and mech
     - Come up with a name and background for your mechaneer
-    - Decide what you want to focus your mech on (heavy weapons, speed, etc.)
+    - Decide what you want to focus your mech on (combat, field repairs, science, etc.)
 2. Choose a Frame
     - Smaller mechs are faster and often don't need auxiliary cooling
     - Larger mechs have much more firepower
-3. Choose your mech's main weapons and attachments
-    - Find a some weapons and equipment that fit your vision
-    - Choose a power type that lets you get most of the weapons and equipment you want
+3. Choose your mech's main weapons and equipment
+    - Find attachments that fit your vision
+    - Choose a power type that lets you get most of the attachments you want
     - Don't worry about filling all your slots here, just pick a couple must-haves
 4. Add the control subsystem
-    - Choose a mainframe that provides enough processing for your weapons and frame, plus a bit extra to spare
+    - Choose a mainframe that provides enough processing for your attachments and frame, plus a bit extra to spare
```

Comment:
For this and other places I would just say "for components selected up to this point" instead of "for attachments and frame) which doesn't include infrastructure components like your cabin etc. that you've selected.

Reply:
Addressed in 050de22